core/src/main/scala/kafka/server/ClientQuotaManager.scala

```diff
@@ -392,24 +394,19 @@ class ClientQuotaManager(private val config: ClientQuotaManagerConfig,
       sensorAccessor.getOrCreate(
         getQuotaSensorName(clientQuotaEntity.quotaId),
         ClientQuotaManagerConfig.InactiveSensorExpirationTimeSeconds,
-        lock, metrics,
-        () => clientRateMetricName(clientQuotaEntity.sanitizedUser, clientQuotaEntity.clientId),
-        () => getQuotaMetricConfig(clientQuotaEntity.quota),
-        () => measurableStat
+        clientRateMetricName(clientQuotaEntity.sanitizedUser, clientQuotaEntity.clientId),
+        Some(getQuotaMetricConfig(clientQuotaEntity.quota)),
+        new Rate
```

Comment:
I thought the indirection was deliberate to avoid creating objects for `get` when sensor already exists.